infrastructure/lambda/template.yaml

```diff
@@ -831,6 +832,31 @@ Resources:
       AliasName: !Sub alias/${AWS::StackName}/auditEventQueueEncryptionKey
       TargetKeyId: !Ref MockAuditEventQueueEncryptionKey
 
+  JWKSBucketRole:
+    Type: "AWS::IAM::Role"
+    Properties:
+      AssumeRolePolicyDocument:
+        Statement:
+          - Action: "sts:AssumeRole"
+            Effect: Allow
+            Principal:
+              Service: apigateway.amazonaws.com
+        Version: 2012-10-17
+      Policies:
+        - PolicyName: AccessJWKSjson
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - "s3:Get*"
+                Resource:
+                  - !Sub "arn:aws:s3:::passport-dev-key-rotation-bucket/jwks.json"
```

Comment:
Due to the bucket naming convention, this will need to be in a mapping to account for the env.


dev - !Sub "arn:aws:s3:::passport-dev-key-rotation-bucket/jwks.json"
build - !Sub "arn:aws:s3:::passport-build-key-rotation-bucket/jwks.json"
...etc

Reply:
Rather than mapping can we not just grab the environment here like
- !Sub "arn:aws:s3:::passport-${Environment}-key-rotation-bucket/jwks.json"